### DISCUSSION TITLE
Phrozen PRZ file format

### DISCUSSION THREAD

**Discussion by msspad** [sn4k3/UVtools#705] [General]
The upcoming Phrozen Sonic Mini 8K S supports the PRZ file format. I couldn't find any reference to PRZ in the UVTools code, so I guess it's not supported yet?

Phrozen advertises the format as "open" and it is supposedly supported by Chitubox, Lychee, VoxelDance Tango https://phrozen3d.com/blogs/guides/a-complete-overview-of-sonic-mini-8k-s-vs-sonic-mini-8k

Anybody knows whether the format supports layer-wise properties such as WaitTimeBeforeCure? Or even more control over acceleration parameters?

**Comment by sn4k3** [comment]
It's told that PRZ is similar to GOO format.
Provide a sample file if you can

**Reply by msspad** [reply]
I asked Phrozen support and they cannot provide any info at this moment. However, they plan to do so once their testing is complete (if I understand their answer correctly).

**Reply by sn4k3** [reply]
It is now suported

**Reply by msspad** [reply]
excellent, thanks. Will test it once the printer arrives.
So they use the file format of Elegoo? the PCB in Sonic Mini 8kS has a Phrozen branding, so it seems they left Chitubox? interesting.

**Comment by oelmekki** [comment]
Is it an other case of 3d printer vendor having no clue (or purposefully abusing) what "open" and "standards" mean, or are there actual specs published somewhere? I couldn't find them if there are.

**Reply by Hazoooka** [reply]
Phrozen branding on the 8S PCB may be due to the consolidation of resources, or that they have made an 'open' motherboard, for say installing an open source firmware... ? However I doubt it.
Would be nice to try install one on 8S.

**Comment by tomres** [comment]
Does **Exposure time finder** with "Multiple Exposures" work with the Mini 8K S? About to purchase this machine and am really hoping for this feature.
